river/ensemble/streaming_random_patches.py

```diff
@@ -46,43 +45,47 @@ class SRPClassifier(base.WrapperMixin, base.EnsembleMixin, base.Classifier):
     lam
         Lambda value for resampling.
     drift_detector
-        Drift detector. If `None`, disables concept drift detection and the
-        background learner.
+        Drift detector.
     warning_detector
-        Warning detector. If `None`, disables the background learner and
-        ensemble members are reset if drift is detected.
+        Warning detector.
+    disable_detector:
+        Option to disable drift detectors:<br/>
+        * If `'off'`, detectors are enabled.<br/>
+        * If `'drift'`, disables concept drift detection and the background learner.<br/>
+        * If `'warning'`, disables the background learner and ensemble members are
+         reset if drift is detected.<br/>
```

Comment:
Just a suggestion. What do you think about using:

* If `'off'`, detectors are enabled.
* If `'all'` *(or maybe '`both`')*, disables concept drift detection and the background learner.
* If `'background'` *(or another short word)*, disables the background learner and ensemble members are reset if drift is detected.

Something along these lines. Maybe change the variable name too. My objective is to make it easier to associate the option with what is going on within the ensemble.

Reply:
In this case, I followed the naming convention from the paper.

![image](https://user-images.githubusercontent.com/17923265/105916821-67eedb80-6096-11eb-9247-36b4b25a76a8.png)

Suggestions to improve the text are welcomed.

Reply:
Let's keep the parameters as they are now. The options are well-documented, so users should not be confused with their usage. My idea was to make the options closer to what they do (e.g., disabling both detectors or none of them), as the variable name refers to disabling stuff. In the end, this might complicate the naming pattern and also deviate from the original paper. Thanks for checking the original manuscript :D